src/heretic/config.py

```diff
@@ -23,6 +23,18 @@ class DatasetSpecification(BaseModel):
 class Settings(BaseSettings):
     model: str = Field(description="Hugging Face model ID, or path to model on disk.")
 
+    tagger_plugin: str = Field(default="keyword", description="Tagger plugin to use.")
```

Comment:
I think we can just name this argument `tagger`, same with the others, since by definition a tagger is always a plugin.